src/config/mongoose_config_parser_toml.erl

```diff
@@ -1232,8 +1232,8 @@ mod_global_distrib_connections([<<"tls">>|_] = Path, V) ->
 -spec format_global_distrib_tls([option()]) -> option().
 format_global_distrib_tls(Opts) ->
     case proplists:lookup(enabled, Opts) of
-        {enabled, false} -> false;
-        _ -> Opts
+        {enabled, true} -> maps:remove(enabled, Opts);
```

Comment:
OMG it's a proplist, not a map!
Needs a correction with a force push :-D

Reply:
Yeah I just noticed it failed, I'll push it with some changes to the `.md` file as well because the example would be wrong right now.